BridgeCareApp/BridgeCare/BridgeCare.csproj

```diff
@@ -297,14 +276,14 @@
     <Content Include="Dependencies\RoadCareGlobalOperations.dll" />
     <Content Include="Dependencies\Simulation.dll" />
     <Content Include="Dependencies\SimulationDataAccess.dll" />
-    <Content Include="Dependencies\CalculateEvaluate.pdb" />
-    <Content Include="Dependencies\DatabaseManager.pdb" />
-    <Content Include="Dependencies\FireSharp.dll" />
-    <Content Include="Dependencies\RestSharp.dll" />
-    <Content Include="Dependencies\RestSharp.xml" />
-    <Content Include="Dependencies\RoadCareGlobalOperations.pdb" />
-    <Content Include="Dependencies\Simulation.pdb" />
-    <Content Include="Dependencies\SimulationDataAccess.pdb" />
+    <Content Include="Dpendencies\CalculateEvaluate.pdb" />
```

Comment:
Somehow the name is changed from `Dependencies` to `Dpendencies`.
Make sure there is no folder in Web API application named `Dpendencies`. Check hidden folders also and check in the physical path.